### PR TITLE
Update `general.md` to include `en@truecase`.

### PR DESCRIPTION
**Pull request by RokeJulianLockhart**
Added information about `en@truecase`.

#### **Relevant:**

1.	https://github.com/darktable-org/darktable/issues/2078#issuecomment-2323294035
1.	https://github.com/darktable-org/darktable/issues/17411#issuecomment-2323288609

### CODE REVIEW

**Comment by elstoc** [comment]
Is there a reason this is marked as draft? Do you still have further work to do on it?

File: content/preferences-settings/general.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ draft: false
 Control the overall look and feel of darktable.
 
 interface language
-: Set the language of the user interface. The system default is marked with an * (needs a restart)
+: Set the language of the user interface. It includes the option to enable sentence case (`en@truecase`) for `en` (currently the `US` variant, and may not apply to subsequent variants). The system default is marked with an * (needs a restart)
```
Comment (elstoc) [inline content/preferences-settings/general.md:11]: I probably wouldn't bother mentioning "variants" of EN at the moment since there's currently only one

**Comment by RokeJulianLockhart** [comment]
> https://github.com/darktable-org/dtdocs/pull/677#issuecomment-2323298469

I predicted that there was a 50% chance you'd say what you did in https://github.com/darktable-org/dtdocs/pull/677#discussion_r1740085037, so I marked it as a draft. When I've remediated that, I'll mark it ready for review.

File: content/preferences-settings/general.md (hunk 1 above)
Comment (RokeJulianLockhart) [reply content/preferences-settings/general.md:11]: @elstoc, I agree. BTW, do you know why I don't see a way to easily incorporate this suggestion, like https://github.com/github/docs/blob/e3a169c100d6542bd8178dd47590f0047739107b/content/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request.md#applying-suggested-changes explains?

**Comment by elstoc** [comment]
> I predicted that there was a 50% chance you'd say what you did in https://github.com/darktable-org/dtdocs/pull/677#discussion_r1740085037, so I marked it as a draft.

Ok, no problem. Generally it's best practice to only mark things as draft if you're currently working on them. Sometimes I'll refrain from reviewing things that are still in draft status. I only reviewed this time because I assumed you were finished.

**Comment by RokeJulianLockhart** [comment]
@elstoc, is https://github.com/darktable-org/dtdocs/pull/677/commits/07d2014f69b706c9dca85cdcd70e846598d08f50#diff-6c3d86dab3e3c7b47a97eafe238a53f8dbddc3845634725ed8e08221c9533f0aR11 adequate?

File: content/preferences-settings/general.md (hunk 1 above)
Comment (RokeJulianLockhart) [reply content/preferences-settings/general.md]: @elstoc, is https://github.com/darktable-org/dtdocs/pull/677/commits/07d2014f69b706c9dca85cdcd70e846598d08f50#diff-6c3d86dab3e3c7b47a97eafe238a53f8dbddc3845634725ed8e08221c9533f0aR11 adequate?

**Comment by elstoc** [comment]
Thanks!

**Pull request merged**